services/core-web/src/components/modalContent/UpdateMinespaceUserModal.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import PropTypes from "prop-types";
+import NewMinespaceUser from "@/components/admin/NewMinespaceUser";
+
+const propTypes = {
+  onSubmit: PropTypes.func.isRequired,
+  title: PropTypes.string,
+};
+
+const defaultProps = {
+  title: "",
+};
+
+export const UpdateMinespaceUserModal = (props) => (
+  <div>
+    <NewMinespaceUser {...props} />
```

Comment:
When clicking the edit/pencil icon on an existing minespace users record the modal secondary header reads "Create Proponent", this should be "Edit Proponent". In addition to that, the Email/BCeID username and Mines fields do not pre-populate with the existing record.